UBUNTU2004-PG12.md

```diff
@@ -0,0 +1,97 @@
+# Configuring PostgreSQL 12 on Ubuntu 20.04
+
+The following notes will help setting up a local instance of PostgreSQL 12 on Ubuntu 20.04.
+
+Note that SSL is configured by default.
+
+## Install PostgreSQL
+
+1. Install Postgresql from the default repositories.
+
+   ```bash
+   sudo apt update
+
+   sudo apt install postgresql postgresql-contrib
+   ```
+
+1. Check connectivity to the db as the postgres user.
+
+   ```bash
+   sudo -i -u postgres
+
+   psql
+
+   postgres=# \conninfo
+   You are connected to database "postgres" as user "postgres" via socket in "/var/run/postgresql" at port "5432".
+   postgres=# \q
+   ```
+
+1. Create a DB user account for your linux account. Replacing `<YOUR_LINUX_USERNAME>` with your linux account name.
+
+   ```bash
+   createuser -s -e -w <YOUR_LINUX_USERNAME>
+
+   exit # exit the postgres account session
+   ```
+
+1. Create a Login DB in a session as your user.
+
+   ```bash
+   createdb $USER
+   ```
+
+1. Check connectivity with via your linux account.
+
+   ```bash
+   $ psql
+
+   postgres=# \conninfo
+   You are connected to database "postgres" as user "postgres" via socket in "/var/run/postgresql" at port "5432".
```

Comment:
Would the user really say "postgres" or your linux username?

(Honestly doesn't matter if not, just curious)

Reply:
In think the default postgres user is postgres, we use that user to create a user for our linux username.

Reply:
Good catch thanks :) Fixed in a5ac77d